tests/FakeItEasy.Tests/Core/FakeManagerTests.cs

```diff
@@ -394,7 +394,7 @@ public void Should_invoke_listener_after_call_has_been_intercepted()
             ProcessFakeObjectCall(manager, interceptedCall);
 
             // Assert
-            A.CallTo(() => listener.OnAfterCallIntercepted(completedCall, selectedRule)).MustHaveHappened();
+            A.CallTo(() => listener.OnAfterCallIntercepted(completedCall)).MustHaveHappened();

```

Comment:
We could remove `selectedRule` altogether. We don't _have to_, but we could.

Reply:
I'm not trying to be pushy, but usually you'd comply or comment after I say something like this, and you did neither. I suspect you missed my late-breaking comment.

Reply:
I'm not sure I understand... I did remove `selectedRule`!

Reply:
Ah, you mean the variable is still there! Sorry, will fix.